schema/tests/data/chapter/negative/mailing-list-invalid.yaml

```
@@ -0,0 +1,15 @@
+    name: Leader Name 1
+  - github: leader-name-2
+    name: Leader Name 2
+level: 2
+mailing-list: https://xyz
+name: Valid Chapter Name
+pitch: A very brief, one-line description of your chapter
+tags:
+  - example-tag-1
+  - example-tag-2
+  - example-tag-3
+type: code
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)